monkey/infection_monkey/network/http_fingerprinter.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from contextlib import closing
+from typing import Dict, Iterable, Optional, Set, Tuple
+
+from requests import head
+from requests.exceptions import ConnectionError, Timeout
+
+from infection_monkey.i_puppet import (
+    FingerprintData,
+    IFingerprinter,
+    PingScanData,
+    PortScanData,
+    PortStatus,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPFingerprinter(IFingerprinter):
+    """
+    Goal is to recognise HTTP servers, where what we currently care about is apache.
```

Comment:
Does this mean that the code needs to be modified for other then apache ?

Reply:
This is a prime example of useless comments. They add no value and are hard to maintain. Each time you add something you need to search the code for comments with "currently" and check if it's still the current situation :D

Reply:
Most of the comments in here are left over from the original implementation. I left them so it would be easier to find your place and compare before and after. I can rework them a bit.